### PR TITLE
[Model Update] Manufacturing Capability 3.0.0

### LINKED ISSUE #525: [Model Update]: Manufacturing Capability
*Resiliency Domain / Manufacturing as a Service*

## Update Reason

The Manufacturing Capability data model is being expanded to include the entities Machine, Tool, and Material, along with their corresponding properties.

https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.manufacturing_capability

## MS1 Criteria

- [x] The model that should be updated exists
- [x] The referenced use case exists
- [x] The potential updates are discussed with all stakeholders
- [x] A decision on versioning was made in consensus with the affected parties [according to the guidelines](https://github.com/eclipse-tractusx/sldt-semantic-models/blob/main/documentation/GOVERNANCE.md#versioning).
- [x] The potential update will be either 
  - [x] backward compatible or 
  - [ ] a (chain of) migration strategy(-ies) from all non-deprecated previous versions to the new model will be developed
- [x] A modeling steward has been assigned

### PR DESCRIPTION
**Pull request by FlorianPatzerC0d3**
## Description
**Added**

- Many new concepts to describe resources, materials, capabilities and processes in more detail
- Possibility to reference a bill of process (BoP) using the BoP identifier from the bill of process aspect model
- A certificate representation
- Concept for semantic references

**Changed**

- Replaced the former process representation by a set to reference bills of processes **Attention:** The reference is built via a property from the [bill of process model](https://github.com/FlorianDue/sldt-semantic-models/tree/bill_of_process), which therefore needs to be included into the workspace in order to validate the model or open in in the Aspect Model Editor.
- Replaced the resource representation by representations for machines and machine tools

 -->

Closes #525

## MS2 Criteria
(to be filled out by PR reviewer)
- [x] the model **validates** with the SAMM SDS SDK in the version specified in the Readme.md of this repository by the time of the MS2 check  (e.g., 'java -jar samm-cli.jar aspect \<path-to-aspect-model\> validate ). The  SAMM CLI is available [here](https://eclipse-esmf.github.io/esmf-developer-guide/tooling-guide/samm-cli.html) and in [GitHub](https://github.com/eclipse-esmf/esmf-sdk/releases/tag/v2.3.2)
- [x] use **Camel-Case** (e.g., "MyModelElement" or "TimeDifferenceGmtId", when in doubt follow https://google.github.io/styleguide/javaguide.html#s5.3-camel-case)
- [x] the identifiers for all model elements **start with a capital letter** except for properties
- [x] the identifier for **properties starts with a small letter**
- [x] all model elements **at least contain the fields "preferred name" and "description"** in English language. The description must be comprehensible. It is not required to write full sentences but style should be consistent over the whole model
- [x] Property and the referenced Characteristic should not have the same name
- [x] the versioning in the URN **follows semantic versioning**, where minor version bumps are backwards compatible and major version bumps are not backwards compatible. 
- [x] use **abbreviations only when necessary** and if these are sufficiently common
- [x] **avoid redundant prefixes in property names** (consider adding properties to an enclosing Entity or even adapt the namespace of the model elements, e.g., instead of having two properties `DismantlerId` and `DismantlerName` use an Entity `Dismantler` with the properties `name` and `id` or use a URN like `io.catenax.dismantler:0.0.1`)
- [x] fields `preferredName` and `description` are not the same
- [x] **`preferredName` should be human readable** and follow normal orthography (e.g., no camel case but normal word separation)
- [x] name of aspect is singular except if it only has one property which is a Collection, List or Set. In theses cases, the aspect name is plural.
- [x] units are referenced from the SAMM unit catalog whenever possible
- [x] **use constraints** to make known constraints from the use case explicit in the aspect model 
- [x] when relying on **external standards**, they are referenced through a **"see"** element
- [x] all properties with an [simple type](https://eclipse-esmf.github.io/samm-specification/2.0.0/datatypes.html) have an example value
- [x] metadata.json exists with status "release"
- [x] generated json schema validates against example json payload
- [x] file RELEASE_NOTES.md exists and contains entries for proposed model changes 
- [x] all contributors to this model are mentioned in copyright header of model file

## MS3 Criteria
(to be filled out by semantic modeling team before merge to main-branch)
- [x] All required reviewers have approved this PR (see reviewers section)
- [x] The new aspect (version) will be implemented by at least one data provider
- [x] The new aspect (version) will be consumed by at least one data consumer
- [x] There exists valid test data
- [x] In case of a new (incompatible) major version to an existing version, a migration strategy has been developed
- [x] The model has at least version '1.0.0'
- [x] If a previous model exists, model deprecation has been checked for previous model
- [x] The release date in the Release Note is set to the date of the MS3 approval

### CODE REVIEW

File: io.catenax.manufacturing_capability/1.0.0/metadata.json [hunk 1]
```diff
@@ -1 +1 @@
-{ "status" : "release"} 
+{ "status" : "deprecated"}
```
Comment (agg3fe) [inline io.catenax.manufacturing_capability/1.0.0/metadata.json:1]: status should be "deprecate"

File: io.catenax.manufacturing_capability/3.0.0/ManufacturingCapability.ttl [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+   samm:preferredName "Capability Constraint Properties"@en;
+   samm:description "Set of constraints restricting the capability."@en;
+   samm:characteristic :PropertySetCharacteristic.
+
+:refersToMaterial a samm:Property;
+   samm:preferredName "Refers to Material"@en;
+   samm:description "A specific material the capability constraint can refer to."@en;
+   samm:characteristic :MaterialCharacteristic.
+
+:sementicReferenceId a samm:Property;
```
Comment (agg3fe) [inline io.catenax.manufacturing_capability/3.0.0/ManufacturingCapability.ttl:260]: correct name should be `semanticReferenceId`. typo in property name. you can consider renaming it if you want.

File: io.catenax.manufacturing_capability/RELEASE_NOTES.md [hunk 3]
```diff
@@ -2,7 +2,19 @@
 
 All notable changes to this model will be documented in this file.
 
-## [Unreleased]
+## [3.0.0]
```
Comment (agg3fe) [inline io.catenax.manufacturing_capability/RELEASE_NOTES.md:5]: you can add today's date here if you want MS3 today.

File: io.catenax.manufacturing_capability/3.0.0/ManufacturingCapability.ttl [hunk 4]
```diff
@@ -0,0 +1,303 @@
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#>.
```
Comment (agg3fe) [inline io.catenax.manufacturing_capability/3.0.0/ManufacturingCapability.ttl:1]: copyright headers are missing

**Review by agg3fe — CHANGES_REQUESTED** [review]
please check my comments

File: io.catenax.manufacturing_capability/3.0.0/ManufacturingCapability.ttl (hunk 4 above)
Comment (FlorianPatzerC0d3) [reply io.catenax.manufacturing_capability/3.0.0/ManufacturingCapability.ttl:16]: Done in https://github.com/eclipse-tractusx/sldt-semantic-models/pull/574/commits/e65ddddf876318f0512a3156859223fecc99b912

File: io.catenax.manufacturing_capability/RELEASE_NOTES.md (hunk 3 above)
Comment (FlorianPatzerC0d3) [reply io.catenax.manufacturing_capability/RELEASE_NOTES.md]: Done in https://github.com/eclipse-tractusx/sldt-semantic-models/pull/574/commits/3b800d833ecc8ab27b0417be2563b0bf48f0420c

**Review by agg3fe — APPROVED** [review]
LGTM

**Pull request merged**